examples/erc1155/src/lib.rs

```diff
@@ -0,0 +1,121 @@
+#![cfg_attr(not(test), no_main, no_std)]
+extern crate alloc;
+
+use alloc::vec::Vec;
+
+use alloy_primitives::{Address, U256};
+use openzeppelin_stylus::{
+    token::erc1155::{extensions::IErc1155Burnable, Erc1155, IErc1155},
+    utils::Pausable,
+};
+use stylus_sdk::{
+    abi::Bytes,
+    prelude::{entrypoint, public, sol_storage},
+};
+
+sol_storage! {
+    #[entrypoint]
+    struct Erc1155Example {
+        #[borrow]
+        Erc1155 erc1155;
+        #[borrow]
+        Pausable pausable;
```

Comment:
I think for simplicity make sense to remove pausable here. Just clean example of `Erc1155`

Reply:
Decided with @bidzyyys to have it within this pr

Reply:
But in this case we need to have e2e tests for pausable implementation. Currently I don't see any of them